src/Common/Uuid/UuidRegistry.php

```diff
@@ -67,7 +68,6 @@ class UuidRegistry
         'procedure_specimen' => ['table_name' => 'procedure_specimen', 'table_id' => 'procedure_specimen_id'],
         'questionnaire_repository' => ['table_name' => 'questionnaire_repository'],
         'questionnaire_response' => ['table_name' => 'questionnaire_response'],
-        'patient_related_persons' => ['table_name' => 'patient_related_persons', 'table_id' => 'pid'],
         'form_history_sdoh' => ['table_name' => 'form_history_sdoh'],
         'users' => ['table_name' => 'users'],
         'care_teams' => ['table_name' => 'care_teams'],
```

Comment:
Missing tables that need to be registered. Can anyone point me to the tables if know off top of head?